src/core/PICA/shader_interpreter.cpp

```diff
@@ -303,7 +312,6 @@ void PICAShader::mova(u32 instruction) {
 	const u32 src = getBits<12, 7>(instruction);
 	const u32 idx = getBits<19, 2>(instruction);
 
-	if (idx) Helpers::panic("[PICA] MOVA: idx != 0");
```

Comment:
Shouldn't this be using getIndexedSource before getSourceSwizzled?

Reply:
Oop you're right. I though loading a value into the address register didn't use the previous address register.